lib/rouge/lexers/pascal.rb

```diff
@@ -7,7 +7,7 @@ class Pascal < RegexLexer
       tag 'pascal'
       title "Pascal"
       desc 'a procedural programming language commonly used as a teaching language.'
-      filenames '*.pas'
+      filenames '*.pas', '*.lpr'
```

Comment:
Could you also add a test to the lexer spec to check the extension? The spec is here:

https://github.com/rouge-ruby/rouge/blob/81b36718a602d37b296301e5e9f4c8b2156b6ba2/spec/lexers/pascal_spec.rb#L10-L12

Here's an example from the Markdown lexer spec so you can see how it should look:

https://github.com/rouge-ruby/rouge/blob/81b36718a602d37b296301e5e9f4c8b2156b6ba2/spec/lexers/markdown_spec.rb#L10-L14

Reply:
Thanks for pointing me to a good example.